### PR TITLE
fix(FOROME-1551): progress bar progress

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-1551

1. Styles are approved by the designer.

### CODE REVIEW

File: src/components/units-list/units-list.tsx [hunk 1]
```diff
@@ -71,6 +71,7 @@ export const UnitsList = observer(
     }
 
     const showLoader = isLoading && !filteredGroups.length
+    const showPgrogressBar = fetchedAmount !== 100
```
Comment (EvgeniySobolev) [inline src/components/units-list/units-list.tsx]: typo in `showProgressBar`

**Comment by EvgeniySobolev** [comment]
I think that appearance of the progress bar mustn't move other elements of a page

**Comment by AndreyUstyumenko** [comment]
In my case it just stays white and not getting changed throughout the loading process.

**Review by EvgeniySobolev — APPROVED** [review]

**Comment by QSdmitrioul** [comment]
We accept that on any page we can have only one progress bar?

**Review by QSdmitrioul — APPROVED** [review]

**Comment by Lapk1n** [comment]
yep

**Pull request merged**